prisma/migrations/20221004170744_added_centre/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- AlterTable
+ALTER TABLE "Member" ADD COLUMN     "centreId" INTEGER;
+
+-- CreateTable
+CREATE TABLE "Centre" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "displayText" TEXT NOT NULL,
+    "displaySequence" INTEGER NOT NULL,
```

Comment:
I couldnt fully understand this field. Is it to order by this field when showing in the ui?

Reply:
displaySequence is used to sort the dropdown if we need to. For example, if we want Nepal to show up at the top of the list.